src/cbmc/symex_coverage.cpp

```diff
@@ -260,11 +262,14 @@ void goto_program_coverage_recordt::compute_coverage_lines(
         for(const auto &cov : c_entry->second)
           std::cerr << cov.second.succ->location_number << '\n';
       }
-      assert(c_entry->second.size()==1 || is_branch);
+      INVARIANT(c_entry->second.size()==1 || is_branch,
+        "Instructions other than branch instructions have exactly 1 successor");
 
       for(const auto &cov : c_entry->second)
       {
-        assert(cov.second.num_executions>0);
+        INVARIANT(cov.second.num_executions>0,
+          // FIXME I don't know what this means
+          "Number of executions most be positive");
```

Comment:
There's a problem with that, in that the `PRECONDITION` according to documentation should only contain unmodified arguments to a function, but I can change it, since it makes it easier to drop the message which appears incorrect as well.

Reply:
You might use a `DATA_INVARIANT`, paraphrasing in the message what I said above (coverage entries can only exist with at least one execution).